tools/mock-handling/src/recordings/dashboards.ts

```diff
@@ -76,20 +73,22 @@ export class DashboardRecording implements IRecording {
 
         const ref = idRef(this.dashboardId);
 
-        const [dashboardWithReferences, alerts] = await Promise.all([
+        const [dashboardWithReferences] = await Promise.all([
             backend.workspace(workspace).dashboards().getDashboardWithReferences(ref),
-            backend.workspace(workspace).dashboards().getDashboardWidgetAlertsForCurrentUser(ref),
```

Comment:
I would rather just comment the code related to alerting as alerting support is coming to panther soon.

Reply:
Same not supported

Reply:
We have git right ?